mainnet/jobs.json

```diff
@@ -78,5 +78,53 @@
     "name": "ethusd_cryptocompare",
     "selector": "[USD]",
     "url": "https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD"
+  },
+  { 
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "sklusd_gemini",
+    "selector": "last",
+    "url": "https://api.gemini.com/v1/pubticker/sklusd"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 0,
+    "name": "sklusd_kucoin",
+    "selector": "data.SKL",
+    "url": "https://api.kucoin.com/api/v1/prices?base=USD&currencies=SKL"
+  },
+  {
+    "weight": 1,
+    "power": 2,
+    "selectorType": 1,
+    "name": "sklusd_coingecko",
+    "selector": "/html/body/div[5]/div[5]/div[1]/div/div[1]/div[3]/div/div[1]/span[1]/span",
```

Comment:
hey @manuelbarbas its preferable to add JSON selectors only (ie, `selectorType: 0`) this is because an XHTML selector can change without notice, reducing its reliability. This job can be replaced by another that uses JSON selector.

Reply:
Hey @yohanelly95 ! Makes total sense, thank you. Just updated